relayer/chains/cosmos/tx.go

```diff
@@ -213,8 +213,11 @@ func (cc *CosmosProvider) broadcastTx(
 		cc.LogFailedTx(rlyResp, err, msgs)
 		return err
 	}
-
-	cc.UpdateFeesSpent(cc.ChainId(), cc.Key(), fees)
+	address, err := cc.Address()
+	if err != nil {
+		return fmt.Errorf("Failed to get relayer bech32 wallet addresss, %w", err)
```

Comment:
style nit: Go convention is to use lower case for error messages. Also the comma is uncommon. I'm surprised a linter isn't catching this.

So

```golang
return fmt.Errorf("failed to get relayer bech32 wallet addresss: %w", err)
```

Reply:
I also typically exclude "failed to" and other words because the error message, as it travels up the stack, often looks like:

```
failed to do X: error doing Y: failed to get Z
```

So I'd opt for this message:

```golang
return fmt.Errorf("get relayer bech32 wallet addresss: %w", err)
```

Reply:
Noted! Yea, this repo is fairly inconsistent with cap : /

Since we switched from returning to logging the error, I kept "failed..."